.mvn/wrapper/MavenWrapperDownloader.java

```diff
@@ -20,98 +20,99 @@
 
 public class MavenWrapperDownloader {
 
-    private static final String WRAPPER_VERSION = "0.5.6";
-    /**
-     * Default URL to download the maven-wrapper.jar from, if no 'downloadUrl' is provided.
-     */
-    private static final String DEFAULT_DOWNLOAD_URL = "https://repo.maven.apache.org/maven2/io/takari/maven-wrapper/"
-        + WRAPPER_VERSION + "/maven-wrapper-" + WRAPPER_VERSION + ".jar";
+  private static final String WRAPPER_VERSION = "0.5.6";
+  /** Default URL to download the maven-wrapper.jar from, if no 'downloadUrl' is provided. */
+  private static final String DEFAULT_DOWNLOAD_URL =
+      "https://repo.maven.apache.org/maven2/io/takari/maven-wrapper/"
+          + WRAPPER_VERSION
+          + "/maven-wrapper-"
+          + WRAPPER_VERSION
+          + ".jar";
 
-    /**
-     * Path to the maven-wrapper.properties file, which might contain a downloadUrl property to
-     * use instead of the default one.
-     */
-    private static final String MAVEN_WRAPPER_PROPERTIES_PATH =
-            ".mvn/wrapper/maven-wrapper.properties";
+  /**
+   * Path to the maven-wrapper.properties file, which might contain a downloadUrl property to use
+   * instead of the default one.
+   */
+  private static final String MAVEN_WRAPPER_PROPERTIES_PATH =
+      ".mvn/wrapper/maven-wrapper.properties";
 
-    /**
-     * Path where the maven-wrapper.jar will be saved to.
-     */
-    private static final String MAVEN_WRAPPER_JAR_PATH =
-            ".mvn/wrapper/maven-wrapper.jar";
+  /** Path where the maven-wrapper.jar will be saved to. */
+  private static final String MAVEN_WRAPPER_JAR_PATH = ".mvn/wrapper/maven-wrapper.jar";
 
-    /**
-     * Name of the property which should be used to override the default download url for the wrapper.
-     */
-    private static final String PROPERTY_NAME_WRAPPER_URL = "wrapperUrl";
+  /**
+   * Name of the property which should be used to override the default download url for the wrapper.
+   */
+  private static final String PROPERTY_NAME_WRAPPER_URL = "wrapperUrl";
 
-    public static void main(String args[]) {
-        System.out.println("- Downloader started");
-        File baseDirectory = new File(args[0]);
-        System.out.println("- Using base directory: " + baseDirectory.getAbsolutePath());
+  public static void main(String args[]) {
+    System.out.println("- Downloader started");
```

Comment:
Maybe use Logger instead of System.out.println?

Reply:
Da hast Du im Prinzip Recht. Allerdings möchte ich den MavenWrapperDownloader.java nicht unbedingt anfassen, da er mir generiert wurde. Das einzige was ich ändern musste, ist die Formatierung.

Reply:
Why do we need this wrapper?

Reply:
I see. Maven Wrapper is a thing if you don't have Maven installed. Do you use the mvnw command to build the project? If I remove the `.mvn` directory and use just `mvn clean install`, it works without creating `.mvn` again. I would strongly suggest to remove the wrapper stuff. How is it part of string-boot? Does the project generator generate the project this way?